### PR TITLE
Separate CRDs into its own helm chart

### PR DESCRIPTION
**Pull request by raulcabello**
fixes kubewarden/helm-charts#35

### CODE REVIEW

File: charts/README.md [hunk 1]
```diff
@@ -38,13 +39,14 @@ deployments. All the options are documented in the configuration section.
 Please refer to the release notes of each version of the helm charts.
```
Comment (viccuad) [inline charts/README.md:39]: Nit: 

```diff
- ## Upgrading the chart
+ ## Upgrading the charts
```

File: charts/README.md [hunk 2]
```diff
@@ -38,13 +39,14 @@ deployments. All the options are documented in the configuration section.
...
-To uninstall/delete the `kubewarden-controller` release use the following
+To uninstall/delete kubewarden-controller and kubewarden-crds use the following
 command:
 
 ```console
 $ helm uninstall -n kubewarden kubewarden-controller
+$ helm uninstall -n kubewarden kubewarden-crds
 ```
 
 The command removes all the Kubernetes components associated with the chart, all
```
Comment (viccuad) [inline charts/README.md:52]: Nit: 

```suggestion
The commands remove all the Kubernetes components associated with the chart, all
```
(there's another one in the following phrase)

**Review by viccuad — APPROVED** [review]
LGTM, thanks!

File: charts/README.md (hunk 1 above)
Comment (raulcabello) [reply charts/README.md:39]: done, thanks! :)

File: charts/README.md (hunk 2 above)
Comment (raulcabello) [reply charts/README.md]: done, thanks! :)

**Review by viccuad — APPROVED** [review]
Merging!

**Pull request merged**